### PR TITLE
Fix no parameters issue with createJob

### PR DESCRIPTION
**Pull request by cdupuis**

### CODE REVIEW

File: test/api-helper/misc/job/createJob.test.ts [hunk 1]
```diff
@@ -143,4 +143,38 @@ describe("createJob", () => {
         assert.deepStrictEqual(result, { id: "123456" });
     });
 
+    it("should not reject invalid parameters", async () => {
```
Comment (ddgenome) [inline test/api-helper/misc/job/createJob.test.ts]: Perhaps "should allow an empty parameter array"

Comment (ddgenome) [reply test/api-helper/misc/job/createJob.test.ts]: I removed auto-merge in case you want to change that.

Comment (cdupuis) [reply test/api-helper/misc/job/createJob.test.ts]: Good point!

**Comment by cdupuis** [comment]
Pull request auto merged by Atomist.

* 1 approved review by @ddgenome
* 2 successful checks

[atomist:generated] [auto-merge:on-approve]

**Pull request merged**